src/backend/web/validation.js

```diff
@@ -98,13 +98,12 @@ const queryValidationRules = [
     .bail(),
 ];
 
-const postsIdParamValidationRules = [
-  param('id', 'Id Length is invalid').isLength({ min: 10, max: 10 }),
-];
+const idParamValidationRules = [param('id', 'Id Length is invalid').isLength({ min: 10, max: 10 })];
```

Comment:
I'm still not sure if using the same rule is the right idea here. Technically the feeds and posts database could be from two different databases and the length of their id could be different. Maybe you can @humphd opinion on this.

Reply:
I think repeating ourselves here is probably OK.  In this case, the `id` is stable across all databases (we generate a hash, and truncate it to 10 characters), but that could change.